scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml

```diff
@@ -36,6 +35,7 @@ apt_packages:
   - "clang-10"
   - "clang-format-10"
   - "unzip"
+  - "python"
```

Comment:
Is python required for this PR? Or is it specifically for the 20.04 support?

Reply:
It's specifically for 20.04; python3 is installed by default, but python isn't. It shouldn't affect 18.04, since it already has python installed.